Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10879,6 +10882,31 @@ def xsiam_api_call_with_retries(
     return response
 
 
+def split_data_to_chunks(data, chunk_size):
+    """
+    Splits a string of data into chunks of a specified size.
+
+    :type data: ``str``
+    :param data: The string of data to split.
+    :type chunk_size: ``int``
+    :param chunk_size: The maximum size of each chunk.
+
+    :return: A list of chunks containing the data.
+    :rtype: ``list``
```

Comment:
It returns an iterable of a list and not a list. I think that `collections.Iterable[list]` might be the right option.
https://www.jetbrains.com/help/pycharm/type-syntax-for-docstrings.html#return-type